receiver/otlpreceiver/otlp.go

```diff
@@ -233,3 +236,9 @@ func (r *otlpReceiver) registerLogsConsumer(lc consumer.Logs) error {
 	}
 	return nil
 }
+
+func handleUnmatchedRequests(resp http.ResponseWriter, req *http.Request) {
+	if req.Header.Get("Content-Type") == "" {
+		resp.WriteHeader(http.StatusUnsupportedMediaType)
+	}
+}
```

Comment:
Same should be done for metrics and logs.

Reply:
@bogdandrutu Sure thing 👍